lib/logstash/inputs/exec.rb

```diff
@@ -80,12 +86,12 @@ def execute(command)
         event["command"] = command
         queue << event
       end
+    rescue StandardError => e
+      @logger.error("Error while running command: #{command}", :e => e, :backtrace => e.backtrace)
     rescue Exception => e
-      @logger.error("Exception while running command", :e => e, :backtrace => e.backtrace)
+      @logger.error("Exception while running command: #{command}", :e => e, :backtrace => e.backtrace)
```

Comment:
use a `key` instead of string interpolation,

```ruby
 @logger.error("Exception while running command", :command => command, :e => e, :backtrace => e.backtrace)
```